include/opt-sched/Scheduler/sched_region.h

```diff
@@ -253,6 +256,8 @@ class SchedRegion {
 
   virtual bool EnableEnum_() = 0;
 
+  virtual bool needsSLIL() = 0;
```

Comment:
```suggestion
  virtual bool needsSLIL() const = 0;
```

We are not nearly const correct, but we should still use `const` when possible

Reply:
did this